src/main/resources/org/jenkinsci/account/Application/index.jelly

```diff
@@ -1,31 +1,18 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:t="/org/jenkinsci/account/taglib">
   <t:layout title="Account">
-    <h1>Account</h1>
-    <p>
+    <h1>Hey ${it.myself.firstName}</h1>
```

Comment:
otherwise you get an error in the logs before you are redirected to the login page:
```suggestion
    <j:if test="${it.isLoggedIn()}">
      <h1>Hey ${it.myself.firstName}</h1>
    </j:if>
```